src/main/java/org/junitpioneer/jupiter/StdIoExtension.java

```diff
@@ -33,7 +33,7 @@
 
 	private static final String SYSTEM_IN_KEY = "StdIo_System_In";
 	private static final String STD_IN_KEY = "StdIo_Std_In";
-	private static final String OUT_KEY = "StdIo_Std_Out";
+	private static final String SYSTEM_OUT_KEY = "StdIo_Std_Out";
```

Comment:
I'd expect the value for this to be `"StdIo_System_Out"` if it's used to store `System.out`.

Reply:
Good point!